libtado/api.py

```diff
@@ -221,6 +221,51 @@ def refresh_auth(self):
     self.refresh_token = response['refresh_token']
     self.access_headers['Authorization'] = 'Bearer ' + self.access_token
 
+  def get_boiler_state(self, authKey):
+    """
+    Parameters:
+      authKey (str|int): Auth code of bridge (from QR sticker; only V3/V3+
+        bridges supported)
+
+    Returns:
+      None when no boiler state data are available, or:
+
+      state (str): installation status of boiler receiver/thermostat
+      deviceWiredToBoiler (dict): type, serial number, protocol etc of
+        receiver/thermostat.
+      bridgeConnected (bool): brigge connection status
+      hotWaterZonePresent (bool): whether controller includes DHW
+      boiler (dict): output temperature (celcius) with timestamp
+
+    Example:
```

Comment:
Rebase from `master` to apply the doc templating in the `get_boiler_state` method:

* Add a description
* Change the result example layout

```python
  def get_boiler_state(self, authKey):
    """
    Add a description here.

    [...]

    ??? info "Result example"
      ```json
      {
        "state": "INSTALLATION_COMPLETED",
        "deviceWiredToBoiler": {
          "type": "BR02",
          "serialNo": "SOME_SERIAL",
          "thermInterfaceType": "OPENTHERM",
          "connected": true,
          "lastRequestTimestamp": "2023-11-18T16:22:01.788Z"
        },
        "bridgeConnected": true,
        "hotWaterZonePresent": false,
        "boiler": {
          "outputTemperature": {
            "celsius": 50.01,
            "timestamp": "2023-11-18T16:29:35.785Z"
          }
        }
      }
      ```
    """
```

The PR will generate a preview RTD link to help you fine tuning the mkdocstring.